plane/src/drone/runtime/docker/mod.rs

```diff
@@ -220,16 +251,6 @@ impl Runtime for DockerRuntime {
                 );
                 Ok(false)
             }
-            Err(bollard::errors::Error::DockerResponseServerError {
-                status_code: 404, ..
-            }) => {
```

Comment:
Moving this check up to the inspect container call because we expect to catch a 404 there first.